### PR TITLE
created script to add SUSI to meilix

### LINKED ISSUE #81: Blending of meilix with SUSI.Al
> Cooperate with the SUSI.AI hardware project to offer a dedicated Kiosk Meilix version with SUSI.AI for Raspis and Magic Mirror

@mariobehling, we went through your feedback and now in need of your help to understand the terms to start working on this.

### PR DESCRIPTION
**Pull request by xeon-zolt**
### Short description
I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message Fixes #81 

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by meets2tarun** [comment]
This will enable susi in meilix?
@xeon-zolt

**Comment by xeon-zolt** [comment]
currently it will add SUSI firefox extension

**Comment by meets2tarun** [comment]
Kindly check if that extension work fine with the firefox build of meilix

File: scripts/adding-susi.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+#
+# author - Harsh Lathwal aka xeon zolt (xeon.harsh@gmail.com)
+
+# adding susi to firefox by adding the extension to the Meilix default settings
+git clone https://github.com/fossasia/susi_firefoxbot
+cd susi_firefoxbot/src
+zip -r susi-firefox.xpi *
+cd ..
+cd ..
+mkdir meilix-default-settings/usr/lib/mozilla-firefox/extensions/
```
Comment (abishekvashok) [inline scripts/adding-susi.sh]: Why do we need another script?

Comment (xeon-zolt) [reply scripts/adding-susi.sh]: it is the script to add SUSI in Meilix so if we want to modify it like adding susi app or chrome extension or want to create a Meilix version without them its easy to modify and find where changes are that's why I created a different script

Comment (abishekvashok) [reply scripts/adding-susi.sh]: You could track changes with diff generated by git I guess.

**Pull request merged**